services/galley/src/Galley/API/Action.hs

```diff
@@ -745,15 +745,15 @@ notifyConversationAction tag quid notifyOrigDomain con lconv targets action = do
     -- For now these users will not be able to join the conversation until
     -- queueing and retrying is implemented.
     let failedNotifies = lefts notifyEithers
+    for_ failedNotifies $
+      logError
```

Comment:
not sure about the log level here.  is this something we're doing elsewhere?  i think for some clients this may be the right log level, others (more hypothetical ones?) will be hit by a lot of noise.  (Yes, this discussion also doesn't belong here, sorry :)

Reply:
I don't think we're consistent when it comes to logging, including something like this.